packages/react-native/README.md

```diff
@@ -138,6 +95,30 @@ npm run android
 
 When the app opens, enter your API key, press "Configure", and then test your setup by using the button to throw an error. 
 
+If you want to test a release (ie prod) build, use 
+```shell
+npm run ios:release
+```
+or
+```shell
+npm run android:release
+```
+
+### Development workflow
+When developing the react-native package and testing against the example projects, you will need to change the dependency in the example project's `package.json`. Rather than `"@honeybadger-io/react-native": "latest"`, you'll want to generate a tarball of the `react-native` package and install it. For example:
```

Comment:
Thanks for putting together this part here, I know it's not pretty but at least we have a guide.